### ISSUE TITLE
Docker is not installed on RHEL during Pharos cluster installation

### ISSUE THREAD

**Issue by panuhorsmalahti** [kontena/pharos-cluster#559]
I tried to install Kontena Pharos Cluster (tried pharos-cluster 1.2.2 and 1.2.3) but the installation (`pharos-cluster up`) failed with the following error:

```
No package docker-1.13.1 available.
```

RHEL7 doesn't by default include docker, but it can be enabled with the following command:
```
sudo subscription-manager repos --enable=rhel-7-server-extras-rpms
```

After I ran this on every Kontena node the installation proceeded (and docker was installed).

Proposed fix: enable the extras repository in the installation script.

**Comment by jakolehm** [comment]
Documentation update: https://github.com/kontena/pharos-docs/pull/66

**Comment by jnummelin** [comment]
fixed in #624
